### PR TITLE
Create endpoint for submit with body parameters variables

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

**Review by fviale — APPROVED** [review]
LGTM
is there no conflicts with the other jobs POST method that produces and consumes application/json ?

**Comment by oanaBiancaSchiopu** [comment]
> LGTM is there no conflicts with the other jobs POST method that produces and consumes application/json ?

no, because they have different parameters

    @POST
    @Path("{path:jobs}")
    @Produces(MediaType.APPLICATION_JSON)
    JobIdData submitFromUrl(@HeaderParam("sessionid") String sessionId, @HeaderParam("link") String url,
            @PathParam("path") PathSegment pathSegment, @Context UriInfo contextInfos) throws JobCreationRestException,
            NotConnectedRestException, PermissionRestException, SubmissionClosedRestException, IOException;

File: rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java [hunk 1]
```diff
@@ -1539,6 +1539,31 @@ JobIdData submitFromUrl(@HeaderParam("sessionid") String sessionId, @HeaderParam
             throws JobCreationRestException, NotConnectedRestException, PermissionRestException,
             SubmissionClosedRestException, IOException;
 
+    /**
+     * Submits a workflow to the scheduler from a workflow URL, creating hence a
```
Comment (mboussaa) [inline rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java:1543]: u have copied/pasted the endpoint description from the previous one, we should explain the difference between the original endpoint and the new one in the description

Comment (oanaBiancaSchiopu) [reply rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java:1543]: the other 2 endpoints have the same description. In fact, they do the same thing, just the parameters are different. I have changed the parameters description:
     * @param jsonBody
     *            a json with the variables of the workflow

Comment (mboussaa) [reply rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java:1543]: ok.thx

**Review by mboussaa — APPROVED** [review]

**Pull request merged**